go.mod

```diff
@@ -6,21 +6,21 @@ require (
 	github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32
 	github.com/go-logr/zapr v1.2.3
 	github.com/google/go-cmp v0.5.7
-	github.com/onsi/ginkgo/v2 v2.1.3
-	github.com/onsi/gomega v1.18.1
+	github.com/onsi/ginkgo/v2 v2.1.4
+	github.com/onsi/gomega v1.19.0
 	github.com/prometheus/client_golang v1.12.1
 	github.com/spf13/pflag v1.0.5
-	github.com/stolostron/go-log-utils v0.1.0
+	github.com/stolostron/go-log-utils v0.1.1
 	github.com/stolostron/go-template-utils/v2 v2.4.0
 	github.com/stretchr/testify v1.7.0
-	k8s.io/api v0.23.3
-	k8s.io/apimachinery v0.23.3
+	k8s.io/api v0.23.5
+	k8s.io/apimachinery v0.23.5
 	k8s.io/client-go v12.0.0+incompatible
 	k8s.io/klog v1.0.0
-	k8s.io/klog/v2 v2.40.1
+	k8s.io/klog/v2 v2.70.1
 	open-cluster-management.io/addon-framework v0.3.0
-	open-cluster-management.io/governance-policy-propagator v0.0.0
-	sigs.k8s.io/controller-runtime v0.11.1
+	open-cluster-management.io/governance-policy-propagator v0.0.0-00010101000000-000000000000
```

Comment:
Why was this trailing part added?

I intentionally set it to `v0.0.0` to make it obvious that we are replacing it below.

Reply:
`go mod tidy` added that -- let me see if I can make it go away.